src/models/stores/serial.ts

```diff
@@ -52,8 +46,15 @@ export class SerialDevice {
   }
 
   public async requestAndSetPort(){
+
+    // Filter any local devices so we only see arduino uno and compatible in choices
+    const filters = [
+      { usbVendorId: 0x2341, usbProductId: 0x0043 },
+      { usbVendorId: 0x2341, usbProductId: 0x0001 }
+    ];
```

Comment:
This is okay for now, but eventually we're going to want to support other serial devices, e.g. Vernier probes, and so we're going to want to pull this kind of information from a registry of supported devices.

Reply:
Yes, makes sense.  Was thinking this for this very-particular first use case.